index.html

```diff
@@ -85,6 +86,176 @@ <h1>Hey There. I’m Akhtar</h1>
             </li>
         </ul>
     </main>
+    <section class="works">
+        <div class="card card-primary">

```

Comment:
Thank you for the review. <li></li> tags have been added and codes amended wherever appropriate as per request.

Reply:
Great Work :+1:

Reply:
Hi... 
Although this project had already been approved last week, I sent it for review today in a new repository, as I am repeating the week. And this is one of the reviews I got... See attached screenshot.
![192980696_770568687157794_6493254610434792403_n](https://user-images.githubusercontent.com/70698085/121536541-3575c780-ca14-11eb-9663-758f1bb3b9b9.png)

I didn't make the changes asked this time, but instead wrote a comment to ask for clarification as I've been asked to change the 'ul' and 'li' tags that you asked me to use and to put back the 'section' tags. 

Firstly, I am confused that I don't know which is the correct method. And secondly, I'm afraid if I change it and submit it, someone else might ask me to change it back to 'ul' and 'li' tags. If I don't change it, it might not get approved. What is my best approach here?

Thank you for clarifying.